netbout-web/src/main/java/com/netbout/rest/account/TkSaveEmail.java

```diff
@@ -78,7 +80,18 @@
      * @param bse Base
      */
     TkSaveEmail(final Base bse) {
+        this(bse, !Manifests.read("Netbout-Version")
+            .contains("LOCAL"));
+    }
+
+    /**
+     * Ctor.
+     * @param bse Base
+     * @param lcl Local Netbout
+     */
+    TkSaveEmail(final Base bse, final boolean lcl) {
```

Comment:
@mkordas Qulice currently used in Netbout is pretty old. It will reject same names for a parameter and for a field.